src/seedDashboard/seedDashboard.scss

```diff
@@ -278,8 +278,9 @@
           justify-self: flex-start;
         }
         .subdescription {
-          padding-left: 29px;
+          padding-left: 0px;
           padding-right: 29px;
+          padding-top: 16px;
```

Comment:
Why did you add this?  It wasn't called for in the card

Reply:
I have discussed this with @BartuCulha earlier this morning and got his consent to do that.

Reply:
I've added an image and a comment for that in the card, but as I checked it as resolved, now the comment is hidden 😕